build_manpages/build_manpages.py

```diff
@@ -5,8 +5,14 @@
 
 import os
 import shutil
+import sys
 
-from argparse_manpage.compat import ConfigParser
+if sys.version_info >= (3, 11):
+    import tomllib
+else:
+    import tomli as tomllib
+
+from argparse_manpage.compat import ConfigParser, NoSectionError
```

Comment:
I suppressed this warning, as it doesn't seem to make sense to respect it, since the imports seem in the right order; it's just the awkwardness of importing different TOML libraries that causes the odd formatting.

Reply:
Thank you for making the linter happy!
Have you considered the:
```
try:
    import the_modern_variant
except ImportError:
    import the_obsoleted_variant
```
pattern?  This is something I typically prefer (not sure whether it makes pylint happy, too).

Reply:
Sure, I can do that. I copied the code from `pytest`. The change seems to placate pylint, at least locally.